scalalib/src/mill/scalalib/Lib.scala

```diff
@@ -150,7 +150,6 @@ object Lib {
     Util.millProperty("MILL_BUILD_LIBRARIES") match {
       case Some(found) => found.split(',').map(os.Path(_)).distinct.toList
       case None =>
-        millAssemblyEmbeddedDeps
```

Comment:
This line is dead code, but more broadly not sure if this `def resolveMillBuildDeps` method is still correct or whether it or its use sites need to be updated